pkgs/applications/graphics/krita/default.nix

```diff
@@ -5,39 +5,46 @@
 , boost, libraw, fftw, eigen, exiv2, libheif, lcms2, gsl, openexr, giflib
 , openjpeg, opencolorio, vc, poppler, curl, ilmbase
 , qtmultimedia, qtx11extras
-, python3
+, python3Packages
 }:
 
 let
 
 major = "4.1";
-minor = "7";
-patch = "101";
+minor = "8";
+patch = null;
 
 in
 
 mkDerivation rec {
   name = "krita-${version}";
-  version = "${major}.${minor}.${patch}";
+  version = "${major}.${minor}";
 
   src = fetchurl {
     url = "https://download.kde.org/stable/krita/${major}.${minor}/${name}.tar.gz";
-    sha256 = "0pvghb17vj3y19wa1n1zfg3yl5206ir3y45znrgdgdw076m5pjav";
+    sha256 = "0h2rplc76r82b8smk61zci1ijj9xkjmf20pdqa8fc2lz4zicjxh4";
   };
 
-  nativeBuildInputs = [ cmake extra-cmake-modules ];
+  propagatedBuildInputs = [ python3Packages.pyqt5 ];
```

Comment:
Should be sufficient to put this into `buildInputs`.

Reply:
seems to work fine with buildInputs only